src/test/regress/expected/multi_extension.out

```diff
@@ -1198,6 +1235,30 @@ SELECT * FROM multi_extension.print_extension_changes();
 
 -- Snapshot of state at 11.2-1
 ALTER EXTENSION citus UPDATE TO '11.2-1';
+-- verify that the placements are deleted and cleanup records are created
+SELECT * FROM pg_dist_placement ORDER BY shardid;
+ placementid | shardid | shardstate | shardlength | groupid
+---------------------------------------------------------------------
+           1 |  102008 |          1 |           0 |       0
+           3 |  102010 |          1 |           0 |       0
+(2 rows)
+
+SELECT * FROM pg_dist_cleanup;
+ record_id | operation_id | object_type |            object_name            | node_group_id | policy_type
+---------------------------------------------------------------------
+         1 |            0 |           1 | table_with_orphaned_shards_102009 |             0 |           0
+         2 |            0 |           1 | table_with_orphaned_shards_102011 |             0 |           0
+(2 rows)
+
+-- error out as cleanup records remain
+ALTER EXTENSION citus UPDATE TO '11.0-4';
+ERROR:  pg_dist_cleanup is introduced in Citus 11.1
+HINT:  To downgrade Citus to an older version, you should first cleanup all the orphaned resources and make sure pg_dist_cleanup is empty, by executing CALL citus_cleanup_orphaned_resources();
+CONTEXT:  PL/pgSQL function inline_code_block line XX at RAISE
+-- cleanup
+DELETE FROM pg_dist_cleanup;
+DELETE FROM pg_dist_shard WHERE shardid % 2 = 1;
```

Comment:
Afaict this will leave the orphaned shard tables on the node. It seems better to run what the error message suggests:
```
CALL citus_cleanup_orphaned_resources()
```